packages/react/src/context.js

```diff
@@ -20,6 +20,11 @@ let EmotionCacheContext: React.Context<EmotionCache | null> =
 
 export let CacheProvider = EmotionCacheContext.Provider
 
+export let __unsafe_useEmotionCache =
+  function __unsafe_useEmotionCache(): EmotionCache | null {
+    return useContext(EmotionCacheContext)
+  }
+
```

Comment:
```suggestion
export function __unsafe_useEmotionCache(): EmotionCache | null {
    return useContext(EmotionCacheContext)
  }

```

Reply:
I know, this is what I would write but I am trying to respect the repo conventions.  

![image](https://user-images.githubusercontent.com/6702424/127303048-02368a3c-16bf-488f-9c63-a92cab59d7e1.png)

Reply:
I wouldn't really bother about the existing convention here. The main reason why `withEmotionCache` is defined like that is because of the fact that it might be reassigned a few lines below - this doesn't apply to the `__unsafe_useEmotionCache`

Reply:
Ah yes I see!  
However I rebased to do it like this:  
![image](https://user-images.githubusercontent.com/6702424/127511764-8b1d93d0-08c9-4d02-b9de-db90fca0e42d.png)
Because it is meant to be imported like this:  
import { __unsafe_useEmotionCache as useEmotionCache } from "@emotion/react";